Also check if getch(3) returns KEY_ENTER in get_commands(), or it may fail
when the enter key is pressed in some terminals (e.g., TERM=pccon0).

Index: ui.c
--- ui.c.orig
+++ ui.c
@@ -573,6 +573,7 @@ get_commands()
 				  refresh_screen();
 				  break;
 			case 'a': add_item();		break;
+			case KEY_ENTER:
 			case '\r': edit_item(-1);	break;
 			case KEY_DC:
 			case 'd':
